
At leat with dietlibc termios.h is not included thru the other includes of
procps, thus the storage size of the winsize struct is not known in several
files. We just include it globally to make sure it is available in any file.

  - Rene Rebe <rene@exactcode.de>

--- ./proc/procps.h.vanilla	2005-07-28 17:48:01.000000000 +0200
+++ ./proc/procps.h	2005-07-28 17:48:08.000000000 +0200
@@ -109,4 +109,6 @@
 #define HIDDEN_ALIAS(x) extern __typeof(x) x##_direct __attribute__((alias(#x)))
 #endif
 
+#include <termios.h>
+
 #endif
